testsuites/syncgateway/functional/topology_specific_tests/multiple_sync_gateways/test_sg_replicate.py

```diff
@@ -338,8 +376,6 @@ def test_sg_replicate_non_existent_db(params_from_base_test_setup):
 ])
 def test_sg_replicate_push_async(params_from_base_test_setup, num_docs):
 
-    assert num_docs > 0
```

Comment:
why did we remove assert here?

Reply:
This is not required assert. Assert at the begining shouldn't be there. We pass the paramemters with num_docs more than 0.  Looks like it was added mistakenly